service/service.go

```diff
@@ -103,10 +103,28 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
+	var awsHostConfig awsclient.Config
+	{
+		accessKeyID := config.Viper.GetString(config.Flag.Service.AWS.HostAccessKey.ID)
+		accessKeySecret := config.Viper.GetString(config.Flag.Service.AWS.HostAccessKey.Secret)
+
+		if accessKeyID == "" && accessKeySecret == "" {
```

Comment:
This feels more helpful to me. Having all the credentials doubled up seems a bit onerous, considering that hosts and guests being in different accounts is (at least currently) the exception.

Reply:
IMO it should be separate.

Reply:
I think this is helpful and I like that the logging makes it clear if the host cluster creds are being reused.